### DISCUSSION TITLE
Spam texts moved to Junk on iphone remain visible in main Messages screen of linked/synced Mac (Continuity bug?)

### DISCUSSION THREAD

**Discussion by glennzilla** [afterxleep/Bouncer#70] [Q&A] [answered]
This issue may be less with Bouncer itself but more with the underlying iOS continuity feature that syncs Messages between the iPhone and a linked Mac:  when Bouncer moves spam texts to the Junk folder, they disappear from the main Messages display on the iPhone, but they remain visible on the synced Mac version of the app.  Since I usually use Messages while on my Mac, the Bouncer benefits are somewhat lost.  If it's an IOS Continuity bug, then I guess my only hope is to open a bug with Apple and wait years in the dark for them to fix it.  Any suggestions here?

**Comment by alextwigkit** [comment]
I have the same question. Thanks!!

**Comment by afterxleep** [comment, marked as answer]
This is a common issue, but unfortunately, Apple has not implemented this feature.  If you use text forwarding on macOS or iPadOS, you'll still receive the filtered SMS messages there.
